core/model-api/src/main/java/org/eclipse/rdf4j/model/Namespace.java

```diff
@@ -11,9 +11,20 @@
 
 /**
  * A namespace, consisting of a namespace name and a prefix that has been assigned to it.
+ * <p>
+ * <strong>Warning</strong> / In order to ensure interoperability of concrete classes implementing this interface,
+ * {@link #equals(Object)} and {@link #hashCode()} methods must be implemented exactly as described in their specs.
+ * 
  */
 public interface Namespace extends Serializable, Comparable<Namespace> {
 
+	/**
+	 * Gets the prefix of the current namespace. The default namespace is represented by an empty prefix string.
+	 *
+	 * @return prefix of namespace, or an empty string in case of the default namespace.
+	 */
+	public String getPrefix();
```

Comment:
Adding new methods to an existing public interface (without a default implementation) is not backwards compatible.

Reply:
Er, forget I said that, this is obviously not a new method. It was early, before coffee.